### ISSUE TITLE
Declarative Config - Contact summaries use "context" to describe multiple things

### ISSUE THREAD

**Issue by kennsippell** [medic/medic-conf#169]
Cards for declarative config have an input called `modifyContext` which refers to the data which are passed to forms loaded from the contact page.  It also has an input called `context` while is used to describe variables used when translating the card's value.

With the breaking change to move projects to declarative config, it is a good time to change this confusing naming collision in the declarative system.

**Comment by abbyad** [comment]
Yes, this can be rather confusing! Context for a label, vs context for a form is very different.

I'd be in favour of being more even more explicit, eg:
- `variablesForForms`, and `modifyVariableForForms`
- `variablesForTranslations`

That said, we also use `context` in deciding when a form should show, so we'd have to see what makes sense throughout the app and its configurable bits before making that change.
